project_report/tex/theory_and_methods.tex

```diff
@@ -41,6 +41,32 @@ \subsubsection{Optimal Production Speed}
 
 
 \subsection{Methods}
+\subsubsection{Unified Process}
+Unified Process, UP, is a an iterative and incremental software development
+process framework. UP consist of four phases:
+
+\myparagraph{The Inception Phase}
+The inception phase is the first phase in UP, and it is in this phase that it
+is determined whether the system can be developed or not. It is in this phase
+that it is decided which responsibilities the system has and identify important
+requirements and critical risks. 
```

Comment:
It is in this phase it is decided which responsibilities the system has and where important requirements and critical risks are identified.